### ISSUE TITLE
Dr Nim distribute properly

### ISSUE THREAD

**Issue by juancarlospaco** [nim-lang/Nim#15265]
Dr Nim should be properly packaged/distributed like the rest of the Nim tools (nimpretty, etc) for the next stable release,
if the user has Dr Nim ready to use after Nim installation, probably they will report more issues if any,
and contribute more feedback and pull requests to evolve the tool quickly, thats beneficial for Dr Nim.

Right now the end user has to manually compile it using `koch`, not the best UX for new users.
Users likely prefer the tool ready to be productive out-of-the-box, even if that means few megabytes more on a terabyte disk.

:)

**Comment by Araq** [comment]
Easier said than done, external dependencies are hard to ship on-non-Windows systems.

**Comment by juancarlospaco** [comment]
It does not compile statically ?.
Let just ship the Windows binary like Steam does (???).
:P

**Comment by mratsim** [comment]
Isn't the only external dependency Z3? Can you dlopen it and if it fails display an error "brew install z3" or "please use your favorite package manager to install z3"?

**Comment by Araq** [comment]
No, I don't use the package managers' Z3, I git clone my own of a fixed commit. You cannot control the quality of your software when you go with "just use a random crappy outdated version from your broken-by-design Linux package management system"

**Comment by juancarlospaco** [comment]
But thats the point,  ...the code quality and documentation is awesome,
but is a feature thats like 95% done and that 5% is what separates end users from actually using it,
then kinda everything goes to waste, too many things on Nim feel like this, it just needs a final push.

For real that even running it with `wine` is better that new code thats already abandoned of use.
:)

**Comment by Araq** [comment]
IMO it's far from being usable for end users. It's a side-project, will finish it after 1.4 is out.
